### DISCUSSION TITLE
Ignore special characters in 'contains' query?

### DISCUSSION THREAD

**Discussion by hyunwoona** [prisma/prisma#15991] [Q&A]
Hi,

I am doing `contains` query in findMany on brand names.
There are brand names like `Paula's Choice`, which contains a `'`. I want to return that brand even when user types `Paulas Choice`.
Is there a way to ignore the special characters in the data stored in the table?

**Comment by nurul3101** [comment]
Hello @hyunwoona 👋 

For this case you would need to use [Raw Query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) and pass the characters which needs to be ignored (`'` in this case) in the translate function as described in this [StackOverflow Answer](https://stackoverflow.com/questions/37819019/postgresql-query-on-a-text-column-ignoring-special-characters) . Let me know if this solution works for you?

**Reply by pantharshit00** [reply]
👋 Hello @hyunwoona,  did you perhaps have a chance to check @nurul3101's recommendation? Let us know if you still have any questions!
 
 Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
